--- /tmp/tmp.LkwJrWwCH1	2025-05-22 15:55:57.938144289 +0900
+++ /tmp/tmp.4Gu05SrECa	2025-05-22 15:55:57.938144289 +0900
@@ -559,7 +559,6 @@
 
 interface Ethernet1/3
   description test_port_3
-  shutdown
   switchport mode trunk
   switchport trunk native vlan 999
   switchport trunk allowed vlan 100-101
@@ -1118,7 +1117,7 @@
 --------------------------------------------------------------------------------
 Eth1/1        ENLACE_SEGURO      connected trunk     full    1000    10g        
 Eth1/2        Chase was here     connected 1         full    1000    10g        
-Eth1/3        test_port_3        disabled  trunk     auto    auto    10g        
+Eth1/3        test_port_3        connected trunk     full    1000    10g        
 Eth1/4        test_port_4        connected routed    full    1000    10g        
 Eth1/5        SPINE1 uplink      connected routed    full    1000    10g        
 Eth1/5.4      --                 disabled  routed    auto    auto    10g        
